WEB/Src/PerformanceCollector/PerformanceCollector/QuickPulseTelemetryProcessor.cs

```diff
@@ -126,6 +129,18 @@ public void Initialize(TelemetryConfiguration configuration)
             this.RegisterSelfWithQuickPulseTelemetryModule();
         }
 
+        void IQuickPulseTelemetryProcessor.UpdateGlobalQuotas(Clock timeProvider, float? maxGlobalTelemetryQuota, float? initialGlobalTelemetryQuota, float? quotaAccrualRatePerSec)
+        {
+            if (maxGlobalTelemetryQuota != null || initialGlobalTelemetryQuota != null || quotaAccrualRatePerSec != null)
+            {
+                this.globalQuotaTracker = new QuickPulseQuotaTracker(
```

Comment:
Just checking - you are aware that this.globalQuotaTracker is used concurrently by multiple threads potentially thousands of times every second, correct?